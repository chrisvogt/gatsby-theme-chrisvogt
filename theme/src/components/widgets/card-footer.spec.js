import React from 'react'
import renderer from 'react-test-renderer'
import CardFooter from './card-footer'

describe('CardFooter', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<CardFooter>Test</CardFooter>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
