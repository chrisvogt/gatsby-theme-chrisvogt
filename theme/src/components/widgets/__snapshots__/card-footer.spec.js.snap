// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardFooter matches the snapshot 1`] = `
<div
  className="css-1nldqfs-CardFooter"
>
  Test
</div>
`;
